src/test/app/Offer_test.cpp

```diff
@@ -3340,7 +3340,7 @@ class Offer_test : public beast::unit_test::suite
 
         // The problem was identified when featureOwnerPaysFee was enabled,
         // so make sure that gets included.
-        Env env {*this, features | FeatureBitset(featureOwnerPaysFee)};
+        Env env {*this, features | featureOwnerPaysFee};
```

Comment:
Here is an example of the kind of usage that makes me a little squirmy that we don't supply `+` with the right semantics.  Yes, mathematically, we get the right result.  But there's a cognitive dissonance that I can't use `+` to accomplish the opposite of `-`.

Not a deal breaker.  And we certainly shouldn't change this right now.  And maybe I'll get used to this...

Reply:
We have a couple or operations on our set:
union: `operator|`
intersection: `operator&`
compliment: `operator~`
set difference: `operator-` (`a - b` is (`intersection(a, compliment(b))`)

`operator|` seems like a better choice for union than `operator+`, these are _set_ operations, not arithmetic operations. I wish there were a better choice than `operator-` for set difference, but I can't think of one (`operator\` would work if available).

The best argument I see for allowing `operator+` is in expressions that need to both set and unset features: `all - feat1 + feat2 - feat3` would do the right thing `all - feat1 | feat2 - feat2` not only looks weird, but has the wrong operator precedence. However, we don't actually do that in the code (the vast majority is always either adding or removing features, but not both).

I'd like to live with `operator|` for a while, but yeah, we can revisit adding `operator+` later.